### PR TITLE
Add recommender to AutoML

### LINKED ISSUE #15775: Update ML.NET AutoML docs with the recommender task
The recommender task is now supported in the AutoML API: https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.automl.recommendationexperiment?view=ml-dotnet-preview

These docs need to be updated:
- https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/how-to-use-the-automl-api
- https://docs.microsoft.com/en-us/dotnet/machine-learning/automate-training-with-cli

### PR DESCRIPTION
**Pull request by natke**
Fixes #15775

Preview link: https://review.docs.microsoft.com/en-us/dotnet/machine-learning/automate-training-with-cli?branch=pr-en-us-16328

### CODE REVIEW

File: docs/machine-learning/reference/ml-net-cli-reference.md [hunk 1]
```diff
@@ -276,17 +248,15 @@ Allowed values are:
...
 
-----------------------------------------------------------
-
 ## See also
 
 - [How to install the ML.NET CLI tool](../how-to-guides/install-ml-net-cli.md)
-- [Automate model training with the ML.NET CLI](../automate-training-with-cli.md)
-- [Tutorial: Auto generate a binary classifier using the ML.NET CLI](../tutorials/mlnet-cli.md)
+- [Overview of the ML.NET CLI](../automate-training-with-cli.md)
+- [Tutorial: Auto generate a binary classifier using the ML.NET CLI](../tutorials/sentiment-analysis-cli.md)
```
Comment (luisquintanilla) [inline docs/machine-learning/reference/ml-net-cli-reference.md]:
```suggestion
- [Tutorial: Analyze sentiment using the ML.NET CLI](../tutorials/sentiment-analysis-cli.md)
```

**Review by luisquintanilla — COMMENTED** [review]
Looks good. Few suggestions.

**Comment by luisquintanilla** [comment]
Adding a few more suggestions not reflected in the changes. 

1. how-to-use-the-automl-api, automate-training-with-cli, and ml-net-cli-reference articles all have lists with tasks you can perform. All of them are missing Recommendation. Make sure that Recommendation is part of these docs.
2. Would it make sense to change the filename of ml-net-cli-reference to ml-net-autotrain-cli-reference since the article is specific to autotrain? Also, change the name in the TOC as well.

**Comment by natke** [comment]
Thanks for the review and suggestions. The CLI does not currently support recommendation :-( I've added all of your other suggestions

**Comment by luisquintanilla** [comment]
Thanks for adding the changes. One minor thing. The article https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/how-to-use-the-automl-api?branch=pr-en-us-16328
is missing recommendation from the list of supported tasks. 

image

**Review by luisquintanilla — APPROVED** [review]
Nicely done! Approving pending the comment below.

https://github.com/dotnet/docs/pull/16328#issuecomment-570348316

**Pull request merged**